src/paper_feeds/app.py

```diff
@@ -41,13 +41,18 @@ async def index(request: Request):
     return templates.TemplateResponse('pages/index.html', {"request": request})
 
 @app.post('/search')
-async def search(request: Request, search: Annotated[str, Form()]):
+async def search(request: Request,
```

Comment:
we will probably want to separate this out from crossref service as we add more data providers, but that's not for this PR :)